money.rb

```diff
@@ -0,0 +1,215 @@
+
+puts " ______                                       __   _______ __ "
+puts "|   __ \  .-----.----.-----.-----.-----.---.-.|  | |    ___|__|.-----.---.-.-----.----.-----."
+puts "|    __/ |  -__|   _|__ --|  _  |     |  _  ||  | |    ___|  ||     |  _  |     |  __|  -__|"
+puts "|___|    |_____|__| |_____|_____|__|__|___._||__| |___|   |__||__|__|___._|__|__|____|_____|"
+  
+require "pry"
+class Menu
+	def initialize(app)
+		@app = app
+	end
+  def display
+    while 1
+    	puts ""
```

Comment:
You could add "\n" at the end of the next `puts` statement to avoid this additional `puts` statement.